Documentation/platforms/risc-v/mpfs/index.rst

```diff
@@ -64,17 +64,18 @@ The following list indicates the state of peripherals' support in NuttX:
 ============   =======  =====
 Peripheral     Support  NOTES
 ============   =======  =====
-GPIO           Yes      
+GPIO           Yes
 MMUART         Yes      Uart mode only
-SPI            Yes      
-I2C            Yes      
+SPI            Yes
+I2C            Yes
 eMMC SD/SDIO   Yes      No PHY training
+USB            Yes
+Ethernet MAC   Yes
 Timers         No
-Watchdog       No       
-RTC            No       
-CAN            No       
-eNVM           No       
-USB            No
```

Comment:
USB should work?

Reply:
https://github.com/apache/incubator-nuttx/pull/5740/files/d90ca07a70fbac0ef05afdacc25b724d1ef26157#diff-cfc2aa7229960754dad201acf4a97c917b8ddf85c5ceba9494c10a1bddb24fe9R72